docs/source/audio/configs.rst

```diff
@@ -0,0 +1,266 @@
+NeMo Audio Configuration Files
+==============================
+This section describes the NeMo configuration file setup that is specific to models in the audio collection.
+For general information about how to set up and run experiments that is common to all NeMo models (e.g. Experiment Manager and PyTorch Lightning trainer parameters), see the :doc:`../core/core` section.
+
+The model section of the NeMo audio configuration files generally requires information about the dataset(s) being used, parameters for any augmentation being performed, as well as the model architecture specification.
+
+Example configuration files for all of the NeMo audio models can be found in the
+`config directory of the examples <https://github.com/NVIDIA/NeMo/tree/stable/examples/audio/conf>`_.
+
+NeMo Dataset Configuration
+--------------------------
+
+Training, validation, and test parameters are specified using the ``model.train_ds``, ``model.validation_ds``, and ``model.test_ds`` sections in the configuration file, respectively.
+Depending on the task, there may be arguments specifying the sample rate or duration of the loaded audio examples. Some fields can be left out and specified via the command-line at runtime.
+Refer to the `Dataset Processing Classes <./api.html#Datasets>`__ section of the API for a list of datasets classes and their respective parameters.
+An example train, validation and test datasets can be configured as follows:
+
+.. code-block:: yaml
+
+  model:
+    sample_rate: 16000
+    skip_nan_grad: false
+
+    train_ds:
+      manifest_filepath: ???
+      input_key: audio_filepath # key of the input signal path in the manifest
+      target_key: target_filepath # key of the target signal path in the manifest
+      target_channel_selector: 0 # target signal is the first channel from files in target_key
+      audio_duration: 4.0 # in seconds, audio segment duration for training
+      random_offset: true # if the file is longer than audio_duration, use random offset to select a subsegment
+      min_duration: ${model.train_ds.audio_duration}
+      batch_size: 64 # batch size may be increased based on the available memory
+      shuffle: true
+      num_workers: 8
+      pin_memory: true
+
+    validation_ds:
+      manifest_filepath: ???
+      input_key: audio_filepath # key of the input signal path in the manifest
+      target_key: target_filepath
```

Comment:
maybe add a comment "# key of the target signal path in the manifest" as in train_ds and test_ds?